modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts

```diff
@@ -90,7 +98,12 @@ export class CalendarWidgetPage {
 			});
 	}
 
-	async addEvent(allDay: boolean, dateEnd: string, title: string) {
+	async addEvent(
+		allDay: boolean,
+		dateEnd: string,
+		publishEvent: boolean,
+		title: string
+	) {
```

Comment:
We should name this method's argument to avoid ugly calls like: `addEvent(true, null, true, null);`.

Reply:
So the function signature would be something like:
```
async addEvent(
  {
    allDay,
    dateEnd,
    publishEvent,
    title,
  }
)
```

And corresponding type declarations.

Reply:
Thanks @cgmonte and @igor-franca ❤️